source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
+
+         * - Contains Clear Linux\* OS native software for cloud.
+           - `ethtool <https://clearlinux.org/software/bundle/ethtool/>`_
+
+         * - Utilities for controlling TCP/IP networking and traffic control.
+           - `iproute2 <https://clearlinux.org/software/bundle/iproute2/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - C++ runtime support.
+           - `libstdcpp <https://clearlinux.org/software/bundle/libstdcpp/>`_
+
+         * - Load and enumerate PKCS#11 modules.
+           - `p11-kit <https://clearlinux.org/software/bundle/p11-kit/>`_
+
+   .. tab:: Game Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - All packages required to build the games bundle.
+           - `games-dev <https://clearlinux.org/software/bundle/games-dev/>`_
+
+         * - Godot game engine editor.
+           - `Godot <https://clearlinux.org/software/flathub/godot/>`_
+
+         * - Real-time game engine (Flatpak).
+           - `Unity Hub <https://clearlinux.org/software/flathub/unity-hub/>`_
+   
+         * - Classic point and click adventure game engine and (Flatpak).
+           - `adventure editor <https://clearlinux.org/software/flathub/adventure-editor/>`_
+
+   .. tab:: Kernel Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Installs kernel, initrd, kernel config, system map; creates a bootloader entry.
+           - `kernel-install <https://clearlinux.org/software/bundle/kernel-install/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in LTS kernel.
+           - `kernel-lts-dkms <https://clearlinux.org/software/bundle/kernel-lts-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in native kernel.
+           - `kernel-native-dkms <https://clearlinux.org/software/bundle/kernel-native-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in AWS kernel.
+           - `kernel-aws-dkms <https://clearlinux.org/software/bundle/kernel-aws-dkms/>`_
+
+         * - Run the Kernel-based Virtual Machine (KVM) specific kernel.
+           - `kernel-kvm <https://clearlinux.org/software/bundle/kernel-kvm/>`_
+
+         * - Run the Intel "PK" kernel and enterprise-style kernel with backports.
+           - `kernel-pk <https://clearlinux.org/software/bundle/kernel-pk/>`_
+
+         * - Run the Intel "PK" kernel, IoT 2019, an enterprise-style kernel with backports.
+           - `kernel-iot-lts2019 <https://clearlinux.org/software/bundle/kernel-iot-lts2019/>`_
+      
+         * - Linux Test Project.
+           - `ltp <https://clearlinux.org/software/bundle/ltp/>`_
+
+   .. tab:: Embedded Systems Developer
```

Comment:
I'd put all the `maker-*` bundles in a "Maker" section. The areas don't necessarily overlap well, nor fit into another category well. Looks like we should maybe add a `maker-electronics` bundle with at least `fritzing-app` and `arduino-ide` in it, though.

Reply:
Great suggestion on the maker profile. I was struggling with whether these bundles were a good fit for Embedded Systems. Which now raises the question: Do you agree that Embedded Systems should be omitted? Unless someone else can advise otherwise, I don't see enough related bundles to justify its inclusion.

Reply:
Yeah, I can't think of any dedicated tools we provide for embedded systems development, so this section can be omitted.

Reply:
Cool. Done.